src/FSharpSource.msbuildreferences.targets

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I do think this could be added to the end of FSharpSource.targets, but I don't mind too much either way